README.md

```diff
@@ -1 +1,81 @@
+# Global GitHub Configuration for organization repositories
 
+## Synchronization
+
+### Labels
+
+This repository contains the configuration that all repositories underneath the Chocolatey and Chocolatey Community organization is expected to use.
+The configuration file is located in the `.github/labels.yml` file and contains information about the name of the label, the color, the description and any aliases that the label may be known underneath or previously used.
+At a minimum when adding a new label the name, color and description should be added. Aliases should only be added if the label is being renamed from something else, or known to conflict with a different name.
+
+Removing labels should be avoided at any cost unless it is known that no other repository have an issue or PR that makes use of the label, instead if it is absolutely needed to remove the label two approaches may be used:
+
+- Add a new label specifically for removed labels, and add the label you wish to remove as an alias.
+- Or add/update the local configuration in each of the repositories that make use of the label that will be removed and copy over the existing global value into this local configuration file.
+
+The repositories that synchronizes labels will have their own workflow file called `label-sync.yml` located in the <.github/workflows> directory, repositories that do not contain this file is not being synchronized with the labels
+configured in this repository.
+
+The synchronization will happen automatically at midnight UTC, but can also be invoked manually be navigating to <actions/workflows/label-sync.yml> and press the `Run workflow`. By default this will use the default branch of the repository
```

Comment:
```suggestion
The synchronization will happen automatically at 00:00 UTC, but can also be invoked manually be navigating to <actions/workflows/label-sync.yml> and press the `Run workflow`. By default this will use the default branch of the repository
```